src/main.rs

```diff
@@ -262,20 +273,48 @@ fn exec(cmd: Command) -> Result<String> {
                 dest_wasm.display().to_string().bold()
             ))
         }
+        Command::Pack {
+            manifest_path,
+            verbosity,
+            unstable_options,
+        } => {
+            let manifest_path = ManifestPath::try_from(manifest_path.as_ref())?;
+            let metadata_result = cmd::metadata::execute(
+                &manifest_path,
+                verbosity.try_into()?,
+                false,
+                unstable_options.try_into()?,
+            )?;
+            let pack_result = cmd::metadata::execute(
+                &manifest_path,
+                verbosity.try_into()?,
+                true,
+                unstable_options.try_into()?,
+            )?;
+            Ok(format!(
+                "\nYour contract is ready. You can find it here:\n{}
+                \nYour metadata file is ready. You can find it here:\n{}
+                \nYour packed contract is ready. You can find it here:\n{}",
+                pack_result.wasm_file.display().to_string().bold(),
+                metadata_result.metadata_file.display().to_string().bold(),
+                pack_result.metadata_file.display().to_string().bold()
+            ))
+        }
         Command::GenerateMetadata {
             manifest_path,
             verbosity,
             unstable_options,
         } => {
             let manifest_path = ManifestPath::try_from(manifest_path.as_ref())?;
-            let metadata_file = cmd::metadata::execute(
-                manifest_path,
+            let res = cmd::metadata::execute(
+                &manifest_path,
                 verbosity.try_into()?,
+                false,
                 unstable_options.try_into()?,
             )?;
             Ok(format!(
-                "Your metadata file is ready.\nYou can find it here:\n{}",
-                metadata_file.display()
+                "\nYour metadata file is ready.\nYou can find it here:\n{}",
+                res.metadata_file.display().to_string().bold(),
```

Comment:
```suggestion
                "\nYour metadata is ready.\nYou can find it here:\n{}",
                res.metadata_file.display().to_string().bold(),
```

Calling one thing file and the other don't is inconsistent.